smif/sos_model.py

```diff
@@ -261,6 +261,8 @@ def get_data(self, model, timestep):
 
         """
         new_data = {}
+        timestep_idx = self.timesteps.index(timestep)
```

Comment:
note-to-self: This finds the array index (position) of the data associated with a year